src/components/Modal/node/OpenMultipleChannelsModal.tsx

```
@@ -168,9 +176,18 @@ export const OpenMultipleChannelsModal = () => {
 
   const handleImportClick = () => {
+    (document.activeElement as HTMLInputElement).blur();
     fileInputRef.current?.click();
   };
 
+  function handleEnter (event: any) {
+    if (openChannelModal && canOpen && event.key === 'Enter') {
+      console.log('OpenMultipleChannelsModal event');
+      handleAction();
+    }
+  };
```

ESLint

[error] Unnecessary semicolon.

(no-extra-semi)